src/style.css

```diff
@@ -1,3 +1,81 @@
+:root {
+  --default-color-link: #000;
+  --color-active-link: #00acdc;
+  --background-header: #fff;
+  --box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
+}
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+.header {
+  width: 100%;
+  box-sizing: border-box;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+  height: 60px;
+  background: var(--background-header);
+  box-shadow: var(--box-shadow);
+}
+
+.logo-link {
+  width: 40px;
+  height: 40px;
+  margin: 10px 0;
+}
+
+.nav {
+  display: flex;
+}
+
+.nav-list {
+  display: flex;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+.nav-item {
+  display: flex;
+  margin-right: 20px;
+}
+
+.nav-item:last-child {
+  margin-right: 0;
+}
+
+.nav-link {
+  align-items: center;
+  justify-content: center;
+  padding: 23px 0;
```

Comment:
Don't fix container size (if there is no such a requirement). You've set a fixed padding to the .nav-link. You should let the content size dictate it to avoid overflow or accidental scroll bar.